phonenumber_field/widgets.py

```diff
@@ -57,3 +60,27 @@ def value_from_datadict(self, data, files, name):
         values = super(PhoneNumberPrefixWidget, self).value_from_datadict(
             data, files, name)
         return '%s.%s' % tuple(values)
+
+
+class PhoneNumberInternationalFallbackWidget(TextInput):
+    """
+    A Widget that allows a phone number in a national format, but if given
+    an international number will fall back to international format
+    """
+
+    def __init__(self, region=None, attrs=None):
+        if region is None:
+            region = (getattr(settings, 'PHONENUMBER_DEFAULT_REGION', None)
+                      or getattr(settings, 'PHONENUMER_DEFAULT_REGION', None))
```

Comment:
Maybe I'm wrong but I think that part when you assign value to region variable is duplicated.